hudi-common/src/main/java/org/apache/hudi/common/table/view/FileSystemViewManager.java

```diff
@@ -165,6 +167,11 @@ private static HoodieTableFileSystemView createInMemoryFileSystemView(HoodieMeta
       return new HoodieMetadataFileSystemView(metaClient, metaClient.getActiveTimeline().filterCompletedAndCompactionInstants(),
           metadataSupplier.get());
     }
+    if (metaClient.getMetastoreConfig().enableMetastore()) {
+      return (HoodieTableFileSystemView) ReflectionUtils.loadClass("org.apache.hudi.common.table.view.HoodieMetastoreFileSystemView",
```

Comment:
Ditto

Reply:
Please define the “org.xxxx.View” as a static final constant

Reply:
fixed